cmd/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	config "github.com/openebs/zfs-localpv/pkg/config/v1alpha1"
+	service "github.com/openebs/zfs-localpv/pkg/service/v1alpha1"
+	"github.com/openebs/zfs-localpv/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func main() {
```

Comment:
Can we mention if this is the binary that is built & packaged as a Docker image?
Just wanted some code level comments on why, what, etc.

Reply:
added.